src/Stack/Types/CompilerGitBuild.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE FlexibleInstances #-}
+
+module Stack.Types.CompilerGitBuild
+  ( CompilerGitBuild(..)
+  )
+where
+
+import           Stack.Prelude
+import           Data.Aeson.Extended
+import           Data.Text as T
+import           Pantry (Repo)
+
+data CompilerGitBuild = CompilerGitBuild
```

Comment:
See my comment on the docs above, I'm not following why we need more than the URL and presumably repo type (though I'd be OK with hard-coding Git for now given GHC development practices).

Reply:
Done. I have hard-coded Git for now.